flow360/component/simulation/meshing_param/volume_params.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Literal, Optional, Union
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+from flow360.component.simulation.framework.entity_base import EntityList
+from flow360.component.simulation.primitives import Box, Cylinder, Face
+from flow360.component.simulation.unit_system import LengthType
+
+"""
+Meshing settings that applies to volumes.
+"""
+
+
+class UniformRefinement(Flow360BaseModel):
+    entities: EntityList[Box, Cylinder] = pd.Field()
+    spacing: LengthType.Positive = pd.Field()
+
+
+class AxisymmetricRefinement(Flow360BaseModel):
```

Comment:
(1) Refinement
(1.1) Box
(1.2) Cylinder
(a) By default, we only require minimal input, a cylinder with uniform refinement zone will be generated
(b) If the user prescribe additional parameters, then an anisotropic cylindrical refinement zone, could be an annular cylinder, will be generated
(2) Interface
Interface will be an alias of refinement (Chai's suggestion)
(2.1) Cylinder
This will be an alias of Refinement -> Cylinder, with enclosed objects
(2.2) Sphere
This will be an alias of Refinement -> Sphere, with enclosed objects
P.S. Currently we don't support spherical refinement/interface right now, will support this later on (in-house mesher)